README.md

```diff
@@ -4,7 +4,7 @@ Provides an interface to create tracking events which are sent to AWS Kinesis.
 
 ## Requirements
 
-- [Nodejs](https://nodejs.org/en/download/) 12.13 with NPM version 6.12
```

Comment:
I updated the npm version that we are using for bundling, does it need a new major, or minor is ok?

Reply:
I'm not the biggest JS expert but this sounds like the upgrade may force some projects that used to use NPM 6.12 to upgrade to version 8 and this would mean that our change is backward incompatible, so this would be a major version.

Reply:
I would do a major then, so I can drop node v12, which is EOL anyway